mappings/net/minecraft/world/gen/feature/DripstoneClusterFeatureConfig.mapping

```diff
@@ -21,5 +21,7 @@ CLASS net/minecraft/unmapped/C_htmabiqc net/minecraft/world/gen/feature/Dripston
 		ARG 7 density
 		ARG 8 wetness
 		ARG 9 wetnessMean
+		ARG 10 maxDistanceFromCenterAffectingChanceOfDripstoneColumn
+		ARG 11 maxDistanceFromCenterAffectingHeightBias
```

Comment:
...these are pretty bad

Reply:
They were already mapped names, I just added the constructor params